addons/l10n_ma/views/res_partner_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_partner_property_form" model="ir.ui.view">
+        <field name="name">res.partner.form.inherit.l10n_ma</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='vat']" position="after">
+                <field name="company_registry" string="ICE" attrs="{'invisible': [('country_code', '!=', 'MA')]}"></field>
```

Comment:
But isn't company_registry in the view already ? You're adding it a second time ? :thinking: 
Depending what you do here, this might as well be improved in master.